### PR TITLE
Complete IANA Considerations section

### LINKED ISSUE #383: IANA registrations missing for some defined parameters
Some of the parameters defined by this specification that could be registered in IANA registries are not being registered.  For instance, there's no IANA Authorization Server Metadata registrations being performed at all.

Like https://github.com/openid/OpenID4VP/issues/239, we need to do a comprehensive inventory of all new parameters defined, and for those that have applicable IANA registries, add them to the IANA Considerations section.

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #383

### CODE REVIEW

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -2311,6 +2363,19 @@ This specification registers the following media types in the IANA "Media Types"
...
+## Uniform Resource Identifier (URI) Schemes Registry
+
+This specification registers the following URI scheme
+in the IANA "Uniform Resource Identifier (URI) Schemes" registry [@IANA.URI.Schemes].
+
+### openid-credential-offer
+
+* URI Scheme: openid-credential-offer
+* Description: Custom scheme used for credential offers
+* Status: Provisional
```
Comment (jogu) [inline openid-4-verifiable-credential-issuance-1_0.md:2375]: I'm not sure I understand why this is 'Provisional' (especially given the media type registration above is not provisional)?

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 2]
```diff
@@ -2220,76 +2221,127 @@ The following is a non-normative example of a Credential Response containing a C
...
+* Parameter Usage Location: token response
+* Change Controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
+* Reference: (#token-response) of this specification
+
+### c_nonce_expires_in
+
+* Name: `c_nonce_expires_in`
 * Parameter Usage Location: token response
 * Change Controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
 * Reference: (#token-response) of this specification
```
[the remark this answers is not in the document]
Comment (selfissued) [reply openid-4-verifiable-credential-issuance-1_0.md:2290]: I'll watch #381 and update accordingly, when merged.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 1 above)
Comment (selfissued) [reply openid-4-verifiable-credential-issuance-1_0.md]: I've now requested Permanent registration of the URI scheme.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jogu) [reply openid-4-verifiable-credential-issuance-1_0.md:2290]: @selfissued Thanks! #381 is now merged so please update.

[the remark this answers is not in the document]
Comment (selfissued) [reply openid-4-verifiable-credential-issuance-1_0.md]: Registrations for `c_nonce` and `c_nonce_expires_in` removed

**Review by jogu — APPROVED** [review]

**Review by Sakurann — APPROVED** [review]

**Review by tplooker — APPROVED** [review]

**Review by paulbastian — APPROVED** [review]

**Comment by jogu** [comment]
4 approvals, open for 3 weeks and discussed on a few WG calls - merging! Thanks Mike & everyone!

**Pull request merged**